packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -204,6 +205,8 @@ let AsyncComboBox = () => {
 
 describe('ComboBox', function () {
   let user;
+  let testUtilUser = new User();
```

Comment:
The user on line 207 is a separate user from the one that the `testUtilUser` uses, its the original `userEvent` user that the combobox test used historically and is still around since the `testUtilUser` is only used in a subset of the tests. Eventually I'd like to explore customizing the user used in the utils but for the very first version of the util I don't want to expose that level of customizability